gwtp-core/gwtp-mvp-client/src/main/java/com/gwtplatform/mvp/client/PresenterWidget.java

```diff
@@ -225,6 +226,10 @@ public void fireEvent(GwtEvent<?> event) {
         getEventBus().fireEventFromSource(event, this);
```

Comment:
Unfortunately there seems to be no replacement: 

http://www.gwtproject.org/javadoc/latest/com/google/web/bindery/event/shared/package-summary.html